metrics/README.md

```diff
@@ -3,6 +3,20 @@
 This section contains a collection of scripts designed to measure the performance
 and responsiveness of the different components which form Clear Containers.
 
+> **WARNING**
+>
+> Running the metrics tests may modify your existing setup or containers.
+>
+> During initialisation, the metrics tests code tries to bring the system into a
```

Comment:
Lines 10-11 suggested rewrite: During initialization, the metrics tests code tries to bring the system into a known stable state before the tests are executed.